happi/tests/test_loader.py

```diff
@@ -149,3 +150,28 @@ def test_filter_kwargs(item_jinja: OphydItem):
     assert getattr(filtered_dev, 'blank_exclude', 'DNE') == 'DNE'
     for bl in blanks:
         assert getattr(filtered_dev, bl, 'DNE') == 'DNE'
+
+
+class PostDevice:
+    def post_happi_md(self):
+        print('post_happi_md hook run')
```

Comment:
Maybe this could have set an attribute e.g. `self.post_count += 1` and then there wouldn't have been a need to invoke patch/mock in the test cases.